test/expression.txt

```diff
@@ -840,3 +840,10 @@ sum:my_metric_name:rate5m
 
 ==>
 MetricName(MetricIdentifier(Identifier))
+
+# Parsing single function
```

Comment:
Not sure if this test makes sense. I wasn't sure what the issue was at the start so just created a test case. Now proper test is basically to test every single function but at the same time that does not seem like super meningful